File: 프로그래머스/lv1/131705. 삼총사/삼총사.java
class Solution {
    public int solution(int[] number) {
        int answer = 0;
        int l = number.length;
        for(int i = 0; i < l - 2; i++) {
            for(int j = i + 1; j < l - 1; j++) {
                for(int k = j + 1; k < l; k++) {
                    if(number[i] + number[j] + number[k] == 0) {
                        answer++;
                    }
                }
            }
        }
        return answer;
    }
}

File: 프로그래머스/1/131705. 삼총사/삼총사.js
function solution(number) {
    var answer = 0;
    var len = number.length;
    for(let i = 0; i < len - 2; i++) {
        for(let j = i + 1; j < len - 1; j++) {
            for(let k = j + 1; k < len; k++) {
                if((number[i] + number[j] + number[k]) === 0) answer++;
            }
        }
    }
    return answer;
}